build/config/fuchsia/gn_configs.gni

```diff
@@ -25,7 +25,7 @@ declare_args() {
 # value to specify the API level the packages produced from this repository
 # should be targeting, e.g. in their top-level //.gn file. A value of -1
 # means that no API level will be passed to the tools that consumes it.
-fuchsia_target_api_level = 16
+fuchsia_target_api_level = 17
```

Comment:
Any reason not go to 18 directly?

Reply:
I've tested the changes for API 17 on device, and will uprev to 18 once this is landed. I think it's a little safer that way :)